WordPress/twentytwentyone
# Override the link focus style specificity for buttons

Fixes https://github.com/WordPress/twentytwentyone/issues/747

## Summary

The link focus style uses .site a:focus, this specificity is used to make sure that the themes link styles does not affect
the WordPress admin bar.

Because of this specificity, the background color for the focus state was affecting the button block links.
Adding .site to the button block works but it's hacky.

If there is a better solution where the .site is not needed at all, that would be preferred.

## Relevant technical choices:


## Test instructions

This PR can be tested by following these steps:
1. Add a button with a link and a background color. It does not need to be a custom background color, so the filled button will work. Save.
1. View the front and focus on the button link.
1. Confirm that the background color is correct.
1. Change the body background color. 
1. Focus on the link.
1. Confirm that the background color is still correct.


## Quality assurance
